haptics/src/web.ts

```diff
@@ -11,30 +11,38 @@ export class HapticsWeb extends WebPlugin implements HapticsPlugin {
     super({ name: 'Haptics' });
   }
 
-  impact(options: HapticsImpactOptions): void {
+  async impact(options: HapticsImpactOptions): Promise<void> {
     console.log('impact options', options);
+    return Promise.resolve();
```

Comment:
Isn't necessary in an `async` function

```suggestion
```

Reply:
I know it's not needed, but I think it's better to explicitly return the promise to make it more clear and more consistent with iOS/Android code.

But also, I've just noticed that iOS and Android return an empty object (`{}`), instead of undefined, we should be consistent about that too. 
Since the types say `Promise<void>`, I think the native plugins should return undefined instead of the empty object, but it's not possible at the moment, we should make the resolve return that.
Or we should have an `EmptyObject` in core plugins could use to return `Promise<EmptyObject>` instead of `Promise<void>`